### PR TITLE
Feature/unmasked backload

### PR DESCRIPTION
**Pull request by jeffsuperglide**

### CODE REVIEW

File: async_geoprocess/src/cumulus_geoproc/geoprocess/snodas/interpolate.py [hunk 1]
```diff
@@ -198,6 +199,20 @@ async def snodas(cfg: namedtuple, dst: str):
 
         max_dist = 0 if code in no_interp else cfg.max_distance
 
+        # check to see if the file exists
```
Comment (brettpalmberg) [inline async_geoprocess/src/cumulus_geoproc/geoprocess/snodas/interpolate.py]: Suggest not using the approach of trying a S3 HEAD request to see if the file prefix is zz_ or us_. The file prefix of `zz_` or `us_` is known (or can be determined), based on the `dt`.  We only download/process the masked dataset (prefix `us_`) when the unmasked dataset (prefix `zz_`) was not available yet - prior to ~2010.  Recommend modifying the strings.Template for each product to allow us_ or zz_, depending on the date.

Comment (jeffsuperglide) [reply async_geoprocess/src/cumulus_geoproc/geoprocess/snodas/interpolate.py]: I'll take a look at it.

File: async_geoprocess/src/cumulus_geoproc/geoprocess/snodas/interpolate.py [hunk 2]
```diff
@@ -8,8 +8,9 @@
 from string import Template
 
 import pkg_resources
+from botocore.errorfactory import ClientError
 from cumulus_geoproc import logger
-from cumulus_geoproc.configurations import CUMULUS_PRODUCTS_BASEKEY
+from cumulus_geoproc.configurations import CUMULUS_PRODUCTS_BASEKEY, ENDPOINT_URL_S3
```
Comment (brettpalmberg) [inline async_geoprocess/src/cumulus_geoproc/geoprocess/snodas/interpolate.py]: import ENDPOINT_URL_S3 import will not be required after addressing comment below.

**Review by brettpalmberg — CHANGES_REQUESTED** [review]
Changes requested prior to merge.

File: async_geoprocess/src/cumulus_geoproc/geoprocess/snodas/interpolate.py (hunk 1 above)
Comment (brettpalmberg) [reply async_geoprocess/src/cumulus_geoproc/geoprocess/snodas/interpolate.py]: Awesome, thanks @jeffsuperglide

**Review by brettpalmberg — APPROVED** [review]
@jeffsuperglide, this looks good! I think there is still an unused import (comment above), after this minor fix, approved for merge.

File: async_geoprocess/src/cumulus_geoproc/geoprocess/snodas/interpolate.py (hunk 2 above)
Comment (brettpalmberg) [reply async_geoprocess/src/cumulus_geoproc/geoprocess/snodas/interpolate.py]: @jeffsuperglide. I think this import is still present in the latest changes, but unused

**Pull request merged**